www/ebooks/index.php

```diff
@@ -28,6 +28,14 @@
 		$view = null;
 	}
 
+	if($query != ''){
+		$queryStringParams['query'] = $query;
+		// If the user entered a query with the default sort order, change it to relevance sort.
+		if($sort == Enums\EbookSortType::Newest){
```

Comment:
OK, but if the user does a search, then opts to sort by newest, won't this incorrectly force it back to relevance? We should probably add another sort type, `Default`, which would act the same as `Newest`. Or maybe there's a better way to do that.

Reply:
They can't sort by newest after they've done a search because Relevance replaces it. The first sort type is now 

* `S.E. release date` if the query is empty  
* `Relevance` if a query is present

That acts like the Default you described. I tried it out and thought it was natural. One drawback is that you can't enter a query AND sort by newest (even though you can sort by author name, reading ease, and length). That is a shame, but there's no way to tell if the user intended to do that instead of wanting to enter a new query and get results by relevance.

Reply:
But we should be able to sort by newest after searching. I think it's reasonable to search for something general like `holmes` and want to sort by newest ebook. That's why I suggested a new `default` sorting enum, to be used internally. It would be interpreted as `newest` if there's no query, and `relevance` if there is a query.

Reply:
That's a good example search. I'll work on default sorting.